maintainers/maintainer-list.nix

```diff
@@ -2669,6 +2669,11 @@
     email = "info+nix@chmist.com";
     name = "karolchmist";
   };
+  kayhide = {
```

Comment:
Do you mind having this as a separate commit?

Reply:
I will do it!

Reply:
Done!